samples/apps/bookinfo/src/build-docker-services.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#
+# Copyright 2017 Istio Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# Set required env vars. Ensure you have checked out the pilot project
+SCRIPTDIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+DOCKER_REPO_ID=linsun
```

Comment:
Not sure we want user-specific repos - at least make it ${DOCKER_REPO_ID:-$(whoami)} or equivalent.

Reply:
That is fine i'll change that in next commit